GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1073,25 +1082,44 @@ internal enum UpdateTargets
         private UpdateTargets _selectedRevisionUpdatedTargets = UpdateTargets.None;
         private void RevisionGridSelectionChanged(object sender, EventArgs e)
         {
+            if(_ShowRevisionInfoNextToRevisionGrid != AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault)
+            {
+                _ShowRevisionInfoNextToRevisionGrid = AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault;
```

Comment:
shouldn't all other references to `AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault` be replaced with `_ShowRevisionInfoNextToRevisionGrid `?

Reply:
This checks if the setup has changed since last time and refreshes the layout if so. It may be possible to always refresh the layout, but then the position etc should be remembered.

Reply:
my questions was about use of  `AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault` in the code. my understanding that all references should be replaced with `_ShowRevisionInfoNextToRevisionGrid` (i.e. gerhardol/gitextensions@4ca3e3c)